src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+const Categories = () => (
+  <input type="checkbox" />
+);
+
+export default Categories;
```

Comment:
- [ ] The Categories should display a button with the **text** "Check status" not the checkbox. This is the project requirement please implement it

Reply:
thank you @roshan-bajgain for reviewing :smile_cat: :smile_cat: 
I have replaced the checkbox with a button.
![image](https://user-images.githubusercontent.com/102757126/185729203-5feb7145-fadc-4f8f-8011-909b47e75019.png)
**Cheers :wine_glass::wine_glass:**